storage/posix/files.go

```diff
@@ -418,6 +429,32 @@ type treeState struct {
 	Root []byte `json:"root"`
 }
 
+// ensureVersion will fail if the compatibility version stored in the state directory
+// is not the expected version. If no file exists, then it is created with the expected version.
+func (s *Storage) ensureVersion(version uint16) error {
+	versionFile := filepath.Join(s.path, stateDir, "version")
+
+	if _, err := os.Stat(versionFile); errors.Is(err, os.ErrNotExist) {
+		klog.V(1).Infof("No version file exists, creating")
+		data := make([]byte, 2)
+		binary.LittleEndian.PutUint16(data, version)
```

Comment:
I think I'd go for [in order] either ASCII decimal (nice for debugging: "what schema version does your ./state/version file say?") or BigEndian (for consistency with the related C2SP specs, while acknowledging that this file obvs isn't covered by the spec).

Reply:
Used ASCII.